src/it/not-newest-java-level/pom.xml

```diff
@@ -14,7 +14,7 @@
     <packaging>hpi</packaging>
     <properties>
         <jenkins.version>2.89</jenkins.version>
-        <java.level>7</java.level>
+        <java.level>8</java.level>
```

Comment:
Contradicts the intention of the IT added in #91. If there is no usable example, better to just delete this IT.